css/style.css

```diff
@@ -15,48 +17,59 @@ body, input {
   margin: 20px;
   border-radius: 30px;
   background-color: #ccc;
-  color: black; }
+  color: black !important; 
```

Comment:
Please increase the specificity of this  rather than using ! important tag while doing dom interactions